godot-codegen/src/util.rs

```diff
@@ -55,6 +55,14 @@ pub fn make_sname_ptr(identifier: &str) -> TokenStream {
     }
 }
 
+fn class_is_server(class_name: &str) -> bool {
+    class_name
+        .trim_end_matches("Extension")
+        .trim_end_matches("2D")
+        .trim_end_matches("3D")
+        .ends_with("Server")
```

Comment:
Also wrong: `UDPServer`

Reply:
oh fair, that currently matches but i can just explicitly list all the classes. it's just a bit unfortunate to need to do that

Reply:
Yeah, it would be better if Godot exposed it as part of the JSON. The current `api_type` field is near-useless...